src/test/java/cz/inventi/kpj/kpjtesting/KpjTestingTest.java

```diff
@@ -10,41 +19,40 @@
  * 4. Change behavior of {@link HelloWorldServiceImpl#echo(String)} to always
  * returns "KPJ Rulez" <br/>
  */
+@SpringBootTest
```

Comment:
Remove it. This test is without Spring. Try to use just Mockito. These tests are faster and you can test isolated code.
```suggestion
```